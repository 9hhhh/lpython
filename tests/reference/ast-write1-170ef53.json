{
    "basename": "ast-write1-170ef53",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/write1.f90",
    "infile_hash": "5489ab0ef56a5dc63218aab5a3bb89b307fd9cb46be7ca782cb816fa",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-write1-170ef53.stdout",
    "stdout_hash": "7a97d42bf6ad0675be18e91de11d4b92513c4e6b9d2473c2842bb46c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}