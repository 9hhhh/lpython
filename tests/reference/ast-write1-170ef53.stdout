(TranslationUnit [(Print () [(Str "a") 2]) (Print () []) (Print () [a]) (Print () [index]) (Print () [(Str "Scalar_in = ") scalar_in]) (Print () [int_src int_given int_expected]) (Print () [max_angle]) (Print () [n n]) (Print () [(FuncCallOrArray trim [prog] [])]) (Print () [(Str "Assertion failed at ") line]) (Print () [(Str "exactly_conservative= ") exactly_conservative]) (Print () [vec]) (Print () [(FuncCallOrArray trim [input_string] []) (Str ": ")]) (Print () [(Str "2")]) (Print () []) (Print "(a, es22.14)" [(Str "Ekin:     ") Ekin])])
